netlify.toml

```diff
@@ -1,3 +0,0 @@
-[build]
-Command = "npm run build && cp example/* dist/"
-Publish = "dist"
```

Comment:
I removed this file because the build settings were overriding the ones needed for the docs build. These settings are set in the UI for the cms-demo site.